### DISCUSSION TITLE
Configuration of daily limit

### DISCUSSION THREAD

**Discussion by nbensaid** [thedevs-network/kutt#615] [Q&A]
Hi there, 
We installed kutt on our server and we are using it via the API key. After 50 api call we are getting the response:"You have reached your daily limit (50). Please wait 24h."
Where can we configure this setting on our installation?

**Comment by poeti8** [comment]
You should update the environment variable `USER_LIMIT_PER_DAY`, here is the reference: https://github.com/thedevs-network/kutt/blob/develop/.example.env#L39

**Reply by victorsfranco** [reply]
Is there a max limit? Example: Can I have some problem if I increase this env value to 5000?

**Comment by victorsfranco** [comment]
Is there a max limit? Example: Can I have some problem if I increase this env value to 5000?

**Reply by poeti8** [reply]
You can set it to whatever you want.

**Comment by alikssp** [comment]
Thank you very much, my friend. You truly saved a Kutt implementation I was working on! I encountered limitations and couldn't find a solution until now. \o/
